CONTRIBUTING.md

```diff
@@ -5,4 +5,4 @@ No open-source project is going to be successful without contributions. After we
 * The [Pull Request](https://help.github.com/articles/using-pull-requests) is targeted at the `master` branch.
 * The code complies with the [Coding Guidelines for C# 3.0, 4.0 and 5.0](https://csharpcodingguidelines.com/)/.
 * The changes are covered by a new or existing set of unit tests which follow the Arrange-Act-Assert syntax such as is used [in this example](https://github.com/fluentassertions/fluentassertions/blob/daaf35b9b59b622c96d0c034e8972a020b2bee55/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33).
-* If the contribution affects the documentation, please update [**the documentation**](https://github.com/fluentassertions/fluentassertions/tree/master/docs), which is published on the [website](https://fluentassertions.com/documentation/).
+* If the contribution affects the documentation, please update [**the documentation**](https://github.com/fluentassertions/fluentassertions/tree/master/docs), which is published on the [website](https://fluentassertions.com/documentation/).
```

Comment:
❓ Is this just a change of line ending?

Reply:
Yeah, I don't know... I first tried to start recording which SDKs I had to install to get the solution to build and test, but I realized that there's no way I could provide an exhaustive list since of course I already had several installed that were presumably being used. I also figured... maybe that's better left to a separate PR.

I will try to eliminate.